README.md

```diff
@@ -14,7 +14,7 @@ Article explaining the demo is available at
 [https://css-tricks.com/native-like-animations-for-page-transitions-on-the-web](https://css-tricks.com/native-like-animations-for-page-transitions-on-the-web)
 
 Live demo at
-[https://page-transitions.com/](https://page-transitions.com/)
```

Comment:
This link is pointing to Sarah's demo, the original inspiration from this project.

Thanks for the docs care but it should not be changed.